index.js

```diff
@@ -1,16 +1,95 @@
 const functions = require('@google-cloud/functions-framework');
-const axios = require("axios");
+const webpush = require('web-push');
+const Knex = require('knex');
 
+webpush.setVapidDetails(
+  process.env.VAPID_SUBJECT || 'mailto:milena@near.org',
```

Comment:
For a future PR, a helpful alternative would be to set defaults in a local properties file